api/references/activation-events.md

```diff
@@ -31,7 +31,9 @@ MetaDescription: To support lazy activation of Visual Studio Code extensions (pl
 - [`onSearch`](/api/references/activation-events#onSearch)
 - [`onStartupFinished`](/api/references/activation-events#onStartupFinished)
 - [`onTaskType`](/api/references/activation-events#onTaskType)
-- [`onTerminalProfile`](/api/references/activation-events#onTerminalProfile)
+- [`onTerminal`](/api/references/activation-events.md#onTerminal)
+    - [`onTerminalProfile`](/api/references/activation-events#onTerminalProfile)
+    - [`onTerminalShellIntegration`](/api/references/activation-events.md#onTerminalShellIntegration)
```

Comment:
We want to merge this in to vnext, provided the activation events are final this release (I think that will be the case).